### DISCUSSION TITLE
Workflow Automation Capabilities for SigNoz?

### DISCUSSION THREAD

**Discussion by talboren** [SigNoz/signoz#3582] [Ideas]
Hey SigNoz community,

As Datadog recently launched their Workflow Automation capability (https://docs.datadoghq.com/service_management/workflows/), I was wondering what would this community think about extending SigNoz with such capabilities as well? Is that something people here need? I hear many users building their own in-house "automation" capabilities (basically just executing some bash/python script based on some events ^^')

Would love to hear your thoughts about it!

**Comment by pranay01** [comment]
thanks for sharing this here @talboren 

Would love to see if more folks are interested in this

**Comment by rbankole** [comment]
Agreed....thinking a use case such as synthetics monitoring, playbooks or the like

**Comment by talboren** [comment]
> Agreed....thinking a use case such as synthetics monitoring, playbooks or the like

So you mean you would like to see some shape of this capability in SigNoz?
